src/v2/view-builder/views/phone/ChallengeAuthenticatorPhoneView.js

```diff
@@ -57,12 +57,19 @@ const Body = BaseForm.extend(Object.assign(
         : loc('mfa.calling', 'login');
       const enterCodeText = loc('oie.phone.verify.enterCodeText', 'login');
       const carrierChargesText = loc('oie.phone.carrier.charges', 'login');
+      let nicknameText = this.model.get('nickname') || '';
+      if (nicknameText.length > 20) {
+        nicknameText = nicknameText.substring(0, 20) + '...';
+      }
+      if (nicknameText !== '') {
+        nicknameText = '(' + nicknameText + ')';
```

Comment:
Should add a space before the ( here, otherwise the phonenumber and nickname will run together.

Reply:
Addressed. Thanks